app/Models/Alerting/Alerts.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/*
+ * Copyright (C) 2016 Neil Lathwood <neil@lathwood.co.uk>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+namespace App\Models\Alerting;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Builder;
+
+/**
+ *
+ */
+class Alerts extends Model
+{
+    /**
+     * Indicates if the model should be timestamped.
+     *
+     * @var bool
+     */
+    public $timestamps = false;
+    /**
+     * The table associated with the model.
+     *
+     * @var string
+     */
+    protected $table = 'alerts';
+    /**
+     * The primary key column name.
+     *
+     * @var string
+     */
+    protected $primaryKey = 'id';
+
+    // ---- Define Reletionships ----
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function device()
+    {
+        return $this->belongsTo('App\Models\Device', 'device_id');
+    }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function rule()
+    {
+        return $this->belongsTo('App\Models\Alerting\Rules', 'rule_id');
+    }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsToMany
+     */
+    public function user()
+    {
+        return $this->belongsToMany('App\Models\User', 'devices_perms', 'device_id', 'user_id');
```

Comment:
This is used in App\Datatables\AlertsDataTable, I've not got it working any other way yet.